articles/10_why_ros2.md

```diff
@@ -3,7 +3,8 @@ layout: default
 title: Why ROS 2.0?
 permalink: articles/why_ros2.html
 abstract:
-  This article captures the reasons for making breaking changes to the ROS API, hence the 2.0.
+  This article captures the reasons for making breaking changes to the ROS API,
+  hence the 2.0.
```

Comment:
Why do this? It seems to be the reverse operation as the other changes in this file.